utils/rpms/daos.spec

```diff
@@ -203,6 +203,8 @@ mkdir -p %{?buildroot}/%{_unitdir}
 install -m 644 utils/systemd/daos-server.service %{?buildroot}/%{_unitdir}
 install -m 644 utils/systemd/daos-agent.service %{?buildroot}/%{_unitdir}
 
+%pre server
```

Comment:
probably need to bump release version for this

Reply:
Done, thanks.